### PR TITLE
imp: refactor import message

### LINKED ISSUE #1488: import: Doesn't remove temporary files when download failed
*DVC version*: 0.23.2

`dvc import 'https://en.wikipedia.org/wiki/Main_Page'`

```
Importing 'https://en.wikipedia.org/wiki/Main_Page' -> '/home/mroutis/tmp/Main_Page'
[##############################] 100% Main_Page
Error: failed to import https://en.wikipedia.org/wiki/Main_Page - could not find an ETag for 'https://en.wikipedia.org/wiki/Main_Page'
```

`ls ~/tmp` -> `Main_Page`

---

`dvc import https://s3-us-west-2.amazonaws.com/dvc-share/get-started/data.xml`

```
Importing 'https://s3-us-west-2.amazonaws.com/dvc-share/get-started/data.xml' -> '/home/mroutis/tmp/data.xml'
[##                            ] 7% data.xml^CError: interrupted by the user

Having any troubles? Hit us up at https://dvc.org/support, we are always happy to help!
```

`ls ~/tmp` -> `data.xml.e5f6cf5a-a919-43bf-949b-dfd3754227f1`

### PR DESCRIPTION
**Pull request by pared**
Fixes #1488 
I think we should change related issue title, since original issue is not a bug.

### CODE REVIEW

File: dvc/command/imp.py [hunk 1]
```diff
@@ -17,6 +17,7 @@ def run(self):
 
             self.project.imp(self.args.url, out)
         except DvcException:
-            logger.error('failed to import {}'.format(self.args.url))
+            logger.error('failed to import {}. You can add it with `dvc add` '
```
Comment (efiop) [inline dvc/command/imp.py]: It seems a little misleading. If import failed, we should probably tell the user that he can download it himself and `dvc add` it as an alternative.

Comment (pared) [reply dvc/command/imp.py]: right

**Comment by efiop** [comment]
@pared I've modified the error message a bit, please take a look. I was thinking it might clarify that "Download it" it is not a call for action, but rather an alternative suggestion. What do you think?

**Comment by pared** [comment]
Yeah, its much better.

**Pull request merged**